zerver/webhooks/github/tests.py

```diff
@@ -479,6 +479,10 @@ def test_ignored_pull_request_actions(self) -> None:
             payload = orjson.dumps(data).decode()
             self.verify_post_is_ignored(payload, "pull_request")
 
+    def test_ignored_pull_request_review_actions(self) -> None:
+        payload = self.get_body("pull_request_review__edited_empty_changes")
+        self.verify_post_is_ignored(payload, "pull_request_review")
+
```

Comment:
Since this test only checks the empty changes edited actions and not all ignored pull request review actions. It's better to name it as `test_pull_request_review_edited_empty_changes_ignore`.